www/js/model.js

```diff
@@ -268,11 +272,32 @@ define([
 					}, this);
 				}
 			},
+			updateLinks: function(/*string*/ id){
+				var nodes = this.getNodes();
+				var removeId = id + "_initial";
```

Comment:
_initial is a string that is defined in model object at the top. Please include it from there. Just see at the top there are getters for it.